.ado/templates/apple-job-javascript.yml

```diff
@@ -8,8 +8,11 @@ steps:
   - script: 'brew bundle'
     displayName: 'brew bundle'
 
-  - script: brew link node@10 --overwrite --force
-    displayName: 'ensure node 10'
+  - script: sudo chown -R $(whoami) /usr/local/*
```

Comment:
Can you update the summary with an explanation of some of these changes? Seems sort of unrelated and I'm not quite understanding what's going on here.